src/views/Upload.vue

```diff
@@ -39,27 +65,131 @@ import UploadTab from '@/components/Upload/UploadTab.vue';
 import UploadFile from '@/components/Upload/UploadFile.vue';
 import UploadText from '@/components/Upload/UploadText.vue';
 
+import axios from 'axios';
+
 export default {
 	components: {
 		Navigation,
 		UploadTab,
 		UploadFile,
-		UploadText,
+		UploadText,	
 	},
 	data: function () {
 		return {
 			selectedTab: '',
 			tabs: ['Nahrať súbory', 'Vložiť text'],
+			files: [],
+			text: '',
+			uploadPercentage: 0,
+			submitted: false,	
+			alert: {
+				message: 'Nezadali ste súbor ani text, ktorý chcete skontrolovať !',
+				type: 'error',
+				duration: 4000,
+			}
 		};
 	},
+	methods: {
+		submitText(){
+			//console.log('text');
+			let formData = new FormData();
+
+			formData.append('file', this.text);
+			formData.append('title','text...');
+
+			if (this.text.length == 0){
+				return this.$store.dispatch('AlertStore/setAlert', this.alert);
+			}
```

Comment:
Why don't we turn the spinner here?

Reply:
The text is not inserted in textarea so I think there should be an error not a spinner.

Reply:
I think after this line... You're calling axios and then disabling the spinner, but you did not enable it before.

Reply:
Hmm weird, I thought I did it, probably I forgot.

Reply:
You did in the other method. (That's why I suggested to merge those methods together, because its very similar)